modules/library/referencing/src/main/resources/org/geotools/referencing/proj/PROJRefinements.txt

```diff
@@ -0,0 +1,116 @@
+################################################################
+# PROJ Strings Refinements
+# PROJ supports more than 7400 EPSG CRS definitions.
+# Some of them have specific fields that cannot be easily
+# inferred by the GeoTools referencing entities.
+# This class refines the buffered PROJ String built through
+# a PROJFormatter based on Referencing IdentifiedObjects
+# inspection, by remapping keys, applying sorting,
+# converting units, appending fixed constant strings
+# to the output.
+
+# Refinements have been identified by comparing proj codes
+# produced by GT against proj codes produced by pyproj.
+
+# proj_generatecodes.py (available in test folder)
+# has been used to generate the proj codes
+# gt_proj_compare.py has been used to check the differences
+# and refine the content
+################################################################
+
+# Global Additions: All of these keys get appended
+# to the final proj string
+global.additions= +no_defs +type=crs
+
+# proj.order: defines the expected order of PROJ strings
+proj.order=proj,R_A,axis,datum,lat_1,lat_0,lat_ts,lat_2,lon_0,alpha,gamma,k_0,k,x_0,y_0,ellps,pm,a,b,rf,units,to_meter,no_defs,type
+
+# UNIT regex/replacement
+# Specify units translation
+unit.regex.1=\\+units=m\\*([0-9.]+)
+unit.replacement.1=+units=m +to_meter=$1
```

Comment:
I would add some documentation as comments in this file, explaining how the definitions are meant to be set up and used.